rollup.config.js

```diff
@@ -12,12 +12,19 @@ const babelCfg = {
     noClassCalls: true,
     noDocumentAll: true,
   },
+  extensions: ['.ts'],
+  include: ['src/**/*'],
   presets: [['@babel/preset-env', { targets: 'ie 11' }]],
```

Comment:
The name and version must be space-separated. You can validate queries with the CLI, e.g.: `$ npx browserslist 'ie 11'`.

Also, perhaps the `defaults` query should be included like `defaults, ie 11` (though I'm not sure if that will make any differences in the built files).

Incidentally, I would recommend moving the queries to the `package.json#browserslist` field for interoperability.

Reply:
`defaults` won’t make a difference here since `ie 11` overrides that query to produce older JavaScript. I’m fine with using the `browserslist` field if @KittyGiraudel, but at the same time I don’t see the need to fragment our config information if we don’t have multiple tools that read a browserslist query.

Reply:
I’m not precious about it. Unless there is an obvious reason to move that into the package file, let’s keep it here.